packages/pharos/src/components/coach-mark/pharos-coach-mark.ts

```diff
@@ -0,0 +1,165 @@
+   * @attr delay
+   * @type {Delay}
+   */
+  @property({ reflect: true })
+  public delay: Delay = 'short';
+
+  /**
+   * Text content for the coach mark header
+   * @attr header
+   * @type {String}
+   */
+  @property({ reflect: true })
+  public header = '';
+
+  /**
+   * Style variant
+   * @attr variant
+   * @type {Variant}
+   */
+  @property({ reflect: true })
+  public variant: Variant = 'light';
+
+  /**
+   * Set minimum width of coach mark
+   * @attr width
+   * @type {number}
+   */
+  @property({ reflect: true })
+  public width = 250;
+
+  constructor() {
+    super();
+    this.setOffset();
+    this.resizeObserver.observe(document.body);
+  }
+
+  private resizeObserver = new ResizeObserver(
+    debounce((entries) => {
+      entries.map(() => {
+        this.setOffset();
+        this.requestUpdate();
+      });
+    }, 100)
+  );
+
+  private setOffset() {
+    const id: string = this.getAttribute('id') || '';
+    const targetElement: Element | null = document.querySelector(`[data-coach-mark="${id}"]`);
+    if (!targetElement) return;
+    const thisRect = this.getBoundingClientRect();
+    const targetRect = targetElement.getBoundingClientRect();
+
+    switch (this.side) {
+      case 'top':
+        this.offsetX = (targetRect.left + targetRect.right) / 2 - thisRect.x;
+        this.offsetY = targetRect.top - thisRect.bottom - this.margin;
+        break;
+      case 'right':
+        this.offsetX = targetRect.right - thisRect.left + this.margin;
+        this.offsetY = (targetRect.top + targetRect.bottom) / 2 - thisRect.y;
+        break;
+      case 'left':
+        this.offsetY = (targetRect.top + targetRect.bottom) / 2 - thisRect.y;
+        this.offsetX = targetRect.left - thisRect.right - this.margin;
+        break;
+      default: // bottom
+        this.offsetX = (targetRect.left + targetRect.right) / 2 - thisRect.x;
+        this.offsetY = targetRect.bottom - thisRect.top + this.margin;
+    }
+  }
+
+  protected override render(): TemplateResult {
+    this.setOffset();
+    return html`
+      <div
+        class="coach-mark ${this.delay && this.delay !== 'none' ? `delay-${this.delay}` : ''}"
+        aria-hidden=${this.hide}
+        role="dialog"
+        aria-labelledby="coach-mark-heading"
+        style="transform:translate(${this.offsetX}px,${this.offsetY}px)"
+      >
+        <div class="coach-mark__wrapper coach-mark-side__${this.side}">
+          <div
+            class="coach-mark__content coach-mark-alignment__${this.alignment}"
+            style="min-width:${this.width}px"
+          >
+            <pharos-button
+              id="close-button"
+              class="coach-mark__close"
+              type="button"
+              variant="${this.variant === 'light' ? 'subtle' : 'overlay'}"
+              icon="close"
+              label="Close"
+              @click="${() => (this.hide = true)}"
+            ></pharos-button>
```

Comment:
This looks really large to me, when looking at the outcome. Do we want a smaller close button? cc @david-corneail

Reply:
@daneah Yes, I agree — we have close-small icon represented in the figma design and we have a close-small icon in the Pharos UI kit for Figma.. however I'm not seeing the close-small icon loaded into storybook.. @henryclong can you see if you can switch to using the close-small icon.. let's touch base when you get a second